velox/exec/fuzzer/AggregationFuzzerBase.h

```diff
@@ -212,7 +212,8 @@ class AggregationFuzzerBase {
   std::vector<RowVectorPtr> generateInputDataWithRowNumber(
       std::vector<std::string> names,
       std::vector<TypePtr> types,
-      const CallableSignature& signature);
+      const CallableSignature& signature,
+      const bool& isKRangeFrame);
```

Comment:
We can use a default value of false here so the changes in `AggregationFuzzer.cpp` are not needed.